### ISSUE TITLE
Consider dropping egde attributes

### ISSUE THREAD

**Issue by jkibele** [jkibele/pyriv#3]
We're currently importing all shapefile attributes as edge attributes in the graph. We're only using 'LengthKM' with the NHD data, but I think I'm not going to even use that with the National Map. ...and it's probably better to not assume we'll have that attribute anyway to make things more flexible. I've already got code to calculate distance edge weights.

So, since I think we're not going to use the attributes, it may make sense to get rid of them. I don't see an easy way to dump them out of the graph, so it might be easier to open the shapefile with geopandas, dump everything aside from geometry, and start from there.

So, basically, either: 1) find a way to drop the edge attributes or 2) write out a temporary version of the shapefile with no attributes and import that. I think option 1 is better if there's a way to do it.
